### PR TITLE
Changelog v0.9

### PR DESCRIPTION
**Pull request by NicolasMahe**

### CODE REVIEW

**Comment by antho1404** [comment]
TODO for the new version:
- [ ] Update JS library
- [ ] Update API documentation
- [ ] Update CLI documentation
- [ ] Add Marketplace documentation
- [ ] Add Wallet documentation

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,11 +4,106 @@
...
+- ([#886](https://github.com/mesg-foundation/core/pull/886)) service: removed Task.GetInputParameter\(\) since it's not used anywhere
+- ([#885](https://github.com/mesg-foundation/core/pull/885)) service: simplify Status\(\)
+- ([#884](https://github.com/mesg-foundation/core/pull/884)) service: simplify Stop\(\) & improve errors
+- ([#903](https://github.com/mesg-foundation/core/pull/903)) service: move some types to service.go
+
 #### Fixed
+
+- ([#771](https://github.com/mesg-foundation/core/pull/771)) fix gRPC stream acknowledgement.
+- ([#772](https://github.com/mesg-foundation/core/pull/772)) Improve command logs errors.
+- ([#820](https://github.com/mesg-foundation/core/pull/820)) Fix container package.
```
[the remark this answers is not in the document]
Comment (NicolasMahe) [reply CHANGELOG.md]: Yes it's a fix of the previous version.
eg: https://github.com/mesg-foundation/core/blob/master/container/network.go

File: CHANGELOG.md [hunk 2]
```diff
@@ -4,11 +4,43 @@
...
+## [v0.9.0](https://github.com/mesg-foundation/core/releases/tag/v0.9.0)
+
+#### Breaking Changes
 
-- ([#535](https://github.com/mesg-foundation/core/pull/535)) Run MESG with MESG! A base logic implemented to have system services for core. In future, core will be split to system services and only act as a proxy to them. System services are placed under `~/.mesg/systemservices`. [\#567](https://github.com/mesg-foundation/core/pull/567).
+- ([#731](https://github.com/mesg-foundation/core/pull/731)) Deterministic service hash. ([#804](https://github.com/mesg-foundation/core/pull/804)). ([#877](https://github.com/mesg-foundation/core/pull/877)).
+- ([#801](https://github.com/mesg-foundation/core/pull/801)) Add Service's hash to reply of Deploy API
+- ([#849](https://github.com/mesg-foundation/core/pull/849)) Use base58 to encode service hash.
+- ([#860](https://github.com/mesg-foundation/core/pull/860)) Separate service's configuration from service's dependencies. ([#880](https://github.com/mesg-foundation/core/pull/880)).
+- ([#866](https://github.com/mesg-foundation/core/pull/866)) Rename service's `volumefrom` property.
```
Comment (NicolasMahe) [inline CHANGELOG.md]:
```suggestion
- ([#866](https://github.com/mesg-foundation/core/pull/866)) Rename service's `volumesfrom` property.
```

**Comment by NicolasMahe** [comment]
Needs to be updated with latest PR merged

**Pull request merged**